packages/next/src/server/dev/hot-reloader-turbopack.ts

```diff
@@ -93,7 +93,8 @@ const isTestMode = !!(
 export async function createHotReloaderTurbopack(
   opts: SetupOpts,
   serverFields: ServerFields,
-  distDir: string
+  distDir: string,
+  resetFetch: () => void
```

Comment:
Does it make sense to be this specific? Would something like `onBeforeReload` etc. work? 

My thinking is that `createHotReloaderTurbopack` is a generic function but that's only an assumption.

Reply:
Decided to keep it as is for now, since the lifecycle currently doesn't quite line up between webpack and turbopack.